pkg/addons/default/assets/coredns-1.27.json

```diff
@@ -0,0 +1,376 @@
+{
+    "apiVersion": "v1",
+    "items": [
+      {
+        "apiVersion": "v1",
+        "kind": "Service",
+        "metadata": {
+          "annotations": {
+            "prometheus.io/port": "9153",
+            "prometheus.io/scrape": "true"
+          },
+          "labels": {
+            "eks.amazonaws.com/component": "kube-dns",
+            "k8s-app": "kube-dns",
+            "kubernetes.io/cluster-service": "true",
+            "kubernetes.io/name": "CoreDNS"
+          },
+          "name": "kube-dns",
+          "namespace": "kube-system"
+        },
+        "spec": {
+          "internalTrafficPolicy": "Cluster",
+          "ipFamilies": [
+            "IPv4"
+          ],
+          "ipFamilyPolicy": "SingleStack",
+          "ports": [
+            {
+              "name": "dns",
+              "port": 53,
+              "protocol": "UDP",
+              "targetPort": 53
+            },
+            {
+              "name": "dns-tcp",
+              "port": 53,
+              "protocol": "TCP",
+              "targetPort": 53
+            }
+          ],
+          "selector": {
+            "k8s-app": "kube-dns"
+          },
+          "sessionAffinity": "None",
+          "type": "ClusterIP"
+        }
+      },
+      {
+        "apiVersion": "v1",
+        "kind": "ServiceAccount",
+        "metadata": {
+          "annotations": {},
+          "labels": {
+            "eks.amazonaws.com/component": "coredns",
+            "k8s-app": "kube-dns"
+          },
+          "name": "coredns",
+          "namespace": "kube-system"
+        }
+      },
+      {
+        "apiVersion": "v1",
+        "data": {
+          "Corefile": ".:53 {\n    errors\n    health\n    kubernetes cluster.local in-addr.arpa ip6.arpa {\n      pods insecure\n      fallthrough in-addr.arpa ip6.arpa\n    }\n    prometheus :9153\n    forward . /etc/resolv.conf\n    cache 30\n    loop\n    reload\n    loadbalance\n}\n"
```

Comment:
please use the more current corefile with ready endpoint and health.lameduck.
this was added for 1.22-1.24 but somehow got missing after 1.25 again.

Reply:
We're currently using a script for generating the corefile. I notice that we've indeed accepted in the past external contributions, that applied the changes you are suggesting, for different versions of corefile. I will move forward and merge the PR as is for now, but we'll discuss internally whether we want to streamline these.

Reply:
thanks for the info. highly appricated.
if i can help somewhere to update those scripts or base corefiles please message me.
i realy would like to have this "solved" for future release. we cant run our production clusters without those settings. 
also those changes are well known and used by plenty of other clusters.

for context:
over 3,5 years ago the kubernetes readme got updated: https://github.com/kubernetes/website/pull/18158
the corefile managed inside the kubernetes repo switched from health to ready endpoint for the readiness probe in kubernetes 1.16 and lameduck got added with kubernetes 1.17
see: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/coredns/coredns.yaml.sed